proposals/2021-05-gravity-bridge-deployment/README.md

```diff
@@ -0,0 +1,174 @@
+
+## How do validators support the Gravity Bridge?
+
+Cosmos Hub validators will run three key software components of the Gravity bridge:
+
+- The Gravity bridge module, integrated into gaiad (the binary that runs the Cosmos Hub)
+- The Gravity bridge Orchestrator
+- A Geth light client or any Ethereum full node implementing the JSON-rpc standard
+
+### Cosmos to Ethereum:
+
+To send transactions from Cosmos to Ethereum, the Gravity Bridge module of a validator's Gaia instance first packages the transaction data, and makes it available on an endpoint. The Orchestrator then signs this data with the validator’s Ethereum key, and submits it as a message to the Ethereum network. The Ethereum signature of each Cosmos validator needs to be assembled and submitted to the Ethereum chain by relayers.
+
+Validators may be slashed if they fail to submit Ethereum signatures within 6000 Cosmos Hub blocks (about eight hours) of their creation.
+
+Validators may also be slashed if they sign a message with their Ethereum key that was not created by the Gravity bridge module.
+
+Gravity bridge has no other slashing conditions.
+
+### Ethereum to Cosmos:
+
+The Orchestrator also monitors the Ethereum chain, submitting events that occur on Ethereum to Cosmos as messages. When more than 66% of the active Cosmos voting power has sent a message observing the same Ethereum event, the Gravity module will take action.
+
+This oracle action will not be incentivized, nor will it be enforced with slashing. If validators making up more than 33% of the staked tokens do not participate in the oracle, new deposits and withdrawals will not be processed until those validators resume their oracle obligations.
+
+The oracle was designed to adhere to the Cosmos Hub validator security model without slashing conditions to ensure that a consensus failure on Ethereum does not affect operation of the Cosmos chain.
+
+### Slashing Conditions Spec
+
+https://github.com/cosmos/gravity-bridge/blob/main/spec/slashing-spec.md
+
+## How does it work?
+
+Gravity consists of 4 parts:
+
+- An Ethereum contract called Gravity.sol
+- The Gravity Cosmos module of Gaia
+- The orchestrator program which is run by Cosmos validators alongside the Gravity module
+- A market of relayers, who compete to submit transactions to Ethereum
+
+### Gravity.sol
+
+The Gravity Ethereum contract is a highly compact and efficient representation of weighted powers voting on Ethereum. It contains an Etheruem key from each Cosmos validator, as well as their voting power. This signer set is continuously updated as validation power changes on Cosmos, ensuring that it matches the current Cosmos validator set.
+
+Sending tokens, or updating the validator set, contained in Gravity.sol requires more than 66% of the total voting power to approve the action. In this way Gravity.sol mirrors Tendermint consensus on the Cosmos chain as closely as possible on Ethereum.
```

Comment:
Minor nit but presumably all reference to 66% is actual 2/3 (ie 0.6666666...) ? We've traditionally just referred to this as `+2/3 of voting power`

Reply:
I've switched it to the proper fraction.